context/app/static/js/components/profile/Summary/Description.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Alert } from 'js/shared-styles/alerts';
+import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+import { useAppContext } from 'js/components/Contexts';
+
+export function ProfileDescription() {
+  const { userEmail } = useAppContext();
+  // ensure that the user only sees this message once per account per device
+  const localStorageKey = `${userEmail}-has-seen-profile-description`;
+
+  const hasSeenProfileDescriptionCookie = Boolean(localStorage.getItem(localStorageKey));
+  const [hasSeenProfileDescription, setHasSeenProfileDescription] = React.useState(hasSeenProfileDescriptionCookie);
+  if (hasSeenProfileDescription) {
+    return null;
+  }
+  const onClose = () => {
+    localStorage.setItem(localStorageKey, 'true');
+    setHasSeenProfileDescription(true);
+  };
```

Comment:
We could pull this into a reusable hook as it seems to be popping up throughout the designs.

Reply:
Good idea - I've created https://hms-dbmi.atlassian.net/browse/CAT-663 to track this.